Source/LuaBridge/Array.h

```diff
@@ -0,0 +1,54 @@
+// https://github.com/vinniefalco/LuaBridge
+//
+// Copyright 2018, Dmitry Tarakanov
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <LuaBridge/detail/Stack.h>
+
+#include <array>
+
+namespace luabridge {
+
+template <class T, std::size_t s>
+struct Stack <std::array <T,s> >
+{
+  static void push (lua_State* L, std::array <T,s> const& array)
+  {
+    lua_createtable (L, static_cast <int> (s), 0);
+    for (std::size_t i = 0; i < s; ++i)
+    {
+      lua_pushinteger (L, static_cast <lua_Integer> (i + 1));
+      Stack <T>::push (L, array [i]);
+      lua_settable (L, -3);
+    }
+  }
+
+  static std::array <T, s> get (lua_State* L, int index)
+  {
+    if (!lua_istable (L, index))
+    {
+      luaL_error (L, "#%d argments must be table", index);
```

Comment:
I guess we should throw because the C API uses jumps which bypass the C++ unwinding mechanism = C++ memory leaks.
If `throw` behaves incorrectly, it'll be a bug.

Reply:
Alright I'll add a runtime error, I'm not sure what to write in the description, so correct me if it's not accurate :)